index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <a class="logo decoration" href="#">My Logo</a>
+            <ul class="nav-items">
+                <li class="list-style"><a class="decoration" href="#">Portfolio</a></li>
+                <li class="list-style"><a class="decoration" href="#">About</a></li>
+                <li class="list-style"><a class="decoration" href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <section class="text-section">
+            <h1>I’m Ibrahim <br/> Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <p class="contact">LET’S CONNECT</p>
+        </section>
+        <ul class="icons list-style">
+            <li><img src="./assets/twitter.svg" alt="Twitter icon" /></li>
+            <li><img src="./assets/linkedin.svg" alt="Linked-In icon" /></li>
+            <li><img src="./assets/medium.svg" alt="Medium icon" /></li>
+            <li><img src="./assets/github.svg" alt="Github icon" /></li>
+            <li><img src="./assets/angellist.svg" alt="Angellist icon" /></li>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the `img` tags are wrapped in an anchor tag `a`. I am pointing this out as optional because the previous reviewer forgot to mention it, but i strongly suggest you implement it